compiler-plugin/src/main/java/io/ballerina/lib/ai/np/compilerplugin/CommonUtils.java

```diff
@@ -17,30 +17,36 @@
  */
 package io.ballerina.lib.ai.np.compilerplugin;
 
+import com.google.gson.JsonArray;
 import io.ballerina.compiler.api.ModuleID;
 import io.ballerina.compiler.api.SemanticModel;
 import io.ballerina.compiler.api.symbols.ModuleSymbol;
 import io.ballerina.compiler.syntax.tree.AnnotationNode;
 import io.ballerina.compiler.syntax.tree.Node;
 import io.ballerina.compiler.syntax.tree.QualifiedNameReferenceNode;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.stream.Collectors;
+
 /**
  * Class containing common constants and functionality.
  *
  * @since 0.3.0
  */
-class Commons {
-    private Commons() {
+public class CommonUtils {
```

Comment:
Unless absolutely necessary, wouldn't mix up changes.

Reply:
We need to make this public, the provider utils need the `GeneratedCode` record